packages/back-end/src/controllers/features.ts

```diff
@@ -190,10 +195,10 @@ export async function getFeaturesPublic(req: Request, res: Response) {
       hashSecureAttributes,
     });
 
-    // Cache for 30 seconds, serve stale up to 1 hour (10 hours if origin is down)
+    // The default is Cache for 30 seconds, serve stale up to 1 hour (10 hours if origin is down)
     res.set(
       "Cache-control",
-      "public, max-age=30, stale-while-revalidate=3600, stale-if-error=36000"
+      `public, max-age=${CACHE_CONTROL_MAX_AGE}, stale-while-revalidate=${CACHE_CONTROL_STALE_IF_ERROR}, stale-if-error=${CACHE_CONTROL_STALE_WHILE_REVALIDATE}`
```

Comment:
`stale-if-error` and `stale-while-revalidate` look swapped here

Reply:
cood catch